ini.go

```diff
@@ -29,7 +29,7 @@ const (
 
 	// Maximum allowed depth when recursively substituing variable names.
 	depthValues = 99
-	version     = "1.48.0"
+	version     = "1.48.0-1"
```

Comment:
Version will be changed after merge, no need to do it in advance.

Reply:
I will roll back this change as requested.